synapseclient/core/retry.py

```diff
@@ -38,10 +38,51 @@
 ]
 
 
-def with_retry(function, verbose=False,
-               retry_status_codes=[429, 500, 502, 503, 504], expected_status_codes=[],
-               retry_errors=[], retry_exceptions=[],
-               retries=DEFAULT_RETRIES, wait=DEFAULT_WAIT, back_off=DEFAULT_BACK_OFF, max_wait=DEFAULT_MAX_WAIT):
+def with_retry_network(function, verbose=False,
+                       retry_status_codes=[429, 500, 502, 503, 504], expected_status_codes=[], retry_errors=[],
+                       retry_exceptions=None, retries=3, wait=1, back_off=2, max_wait=30):
```

Comment:
Doesn't retry_exceptions=None here change the default behavior of the old with_retry (which would only retry on the listed exceptions)?